dev/com.ibm.ws.microprofile.config/src/com/ibm/ws/microprofile/config/sources/DefaultSources.java

```diff
@@ -43,6 +43,20 @@
         sources.add(new SystemConfigSource());
         sources.add(new EnvConfigSource());
 
+        addPropertyConfigSources(classloader, sources);
+
+        return sources;
+    }
+
+    /**
+     * Add resources of name {#link ConfigConstants.CONFIG_PROPERTIES} to a List of sources using
+     * the classloader's loadResources method to locate resources.
+     *
+     * @param classloader
+     * @param sources
+     */
+    public static void addPropertyConfigSources(ClassLoader classloader, ArrayList<ConfigSource> sources) {
+
```

Comment:
I'm clearly in a picky mood ... could this be a `get` method rather than an `add`? i.e.
`public static List<ConfigSource> getPropertyFileConfigSources(ClassLoader classloader)`